workload-checks/typical/cases/http_text_to_http_json/experiment.yaml

```diff
@@ -0,0 +1,22 @@
+description: >
+  Simulates a simple Vector use with one HTTP server source and one HTTP sink.
+  This was added as a proof of concept for the SMP workload checks.
+teams: []
+
+labels: {}
+
+checks:
+  - name: memory_usage
+    description: "Memory usage"
+    bounds:
+      series: rss_bytes
+      # The machine has 12Gb free.
+      upper_bound: 3.5Gb
+
+  - name: cpu_utilization
+    description: "CPU utilization"
+    bounds:
+      series: cpu_percentage
+      # The machine has 8 cores available.
+      lower_bound: 400
```

Comment:
I'd argue for lower bound being 0 and upper bound being something less than all CPUs. We do want the app server this Vector is running alongside to have some CPU to use.

Reply:
Good point! This specific experiment is expected to be CPU-heavy. I will tweak it accordingly in order for these bounds to be more useful.